_episodes/03-skimage-images.md

```diff
@@ -101,6 +101,54 @@ image = skimage.io.imread(fname="chair.jpg")
 ~~~
 {: .python}
 
+> ## Import Statements in Python
+>
+> In Python, the `import` statement is used to load additional functionality
+> into a program. This is necessary when we want our code to do something more
+> specialised, which cannot easily be achieved with the limited set of basic
+> tools and data structures available in the default Python environment.
+>
+> Additional functionality can be loaded as a single function or object,
+> a module defining several of these, or a library containing many modules.
+> You will encounter several different forms of `import` statement.
+>
+>
+> ~~~
+> import skimage                 # form 1, load whole skimage library
+> import skimage.io              # form 2, load skimage.io module only
+> from skimage.io import imread  # form 3, load only the imread function
+> import numpy as np             # form 4, load all of numpy into an object called np
+> ~~~
+> {: .language-python }
+>
+> > ## Further Explanation
+> >
+> > In the example above, form 1 loads the entire `skimage` library into the
+> > program as an object. individual modules of the library are then available
+> > within that object, e.g. to access the `imread` function used
+> > in the example above, you would write `skimage.io.imread()`.
+> >
+> > Form 2 loads only the `io` module of `skimage` into the program. When we run
+> > the code, the program will take less time and use less memory because we will
+> > not load the whole `skimage` library. The syntax needed to use the module
+> > remains unchanged.: to access the `imread` function, we would use the same
+> > function call as given for form 1.
+> >
+> > To further reduce the time and memory requirements for your program,
+> > form 3 can be used to import only a specific function from a library/module.
+> > In the example above, the `imread` function could then be called without
+> > the need for the preceding library and function names, i.e. `imread()`.
```

Comment:
Might be going too much into depth, but I'd suggest mentioning namespaces and that form 3 places the function/object in the current namespace and that there is a possibility of overwriting identically named functions from other libraries/modules. Lest people start using `from <module> import *`...

Reply:
addressed in 915c6c7, though I have avoided using the term _namespace_.